senza/manaus/__init__.py

```diff
@@ -0,0 +1,3 @@
+"""
+Package with high level classes to access aws components
+"""
```

Comment:
Manaus... ![](https://image.spreadshirtmedia.com/image-server/v1/designs/11925763,width=178,height=178,version=1337303176/I-see-what-you-did-there.png)